pkg/airgap/images.go

```diff
@@ -40,6 +40,7 @@ func GetImageURIs(spec *v1beta1.ClusterSpec, all bool) []string {
 		spec.Images.KubeRouter.CNI.URI(),
 		spec.Images.KubeRouter.CNIInstaller.URI(),
 		spec.Images.MetricsServer.URI(),
+		spec.Network.NodeLocalLoadBalancing.EnvoyProxy.Image.URI(),
```

Comment:
This is already added a few lines below. If you want to have all images, independent of the configuration, you can set all to true.

Reply:
yeah, missed that, thank you!